core/modules/css/CssAccess.cc

```diff
@@ -533,6 +534,18 @@ CssAccess::getPartTableParams(std::string const& dbName,
         throw KeyValueError(tableKey + "/partitioning", "one of the sub-keys is not numeric: " +
                             std::string(exc.what()));
     }
+    auto iter = paramMap.find("partitioning/lockInMem");
+    if (iter != paramMap.end()) {
+        params.lockInMem = std::stoi(paramMap["partitioning/lockInMem"]);
+    } else {
+      params.lockInMem = false;
+    }
+    iter = paramMap.find("partitioning/scanSpeed");
+    if (iter != paramMap.end()) {
+        params.scanSpeed = std::stoi(paramMap["partitioning/scanSpeed"]);
+    } else {
+        params.scanSpeed = 0;
+    }
```

Comment:
Actually, see my most recent commit. I didn't like repeating a large block of code so I simplified it. I think it is better now, though a bit less efficient, but these functions are marked "deprecated" anyway. Let me know if you are ok with that simplification. I'll squash it all into one commit later.